explainers/update-rendering.md

```diff
@@ -16,9 +19,12 @@ words, if the website intends to show an element whose contents are currently
 skipped, then skipping work may cause jank when the contents are ultimately
 presented.
 
-### Proposal
-The proposal is to add the following function, which lets the user-agent know
-that the element’s contents should be updated in preparation for display:
+### Proposal: `updateRendering()` JavaScript function
```

Comment:
WDYT of just removing updateRendering() for now, and just say "an imperative equivalent to keepUpdated could be added in the future".

Reply:
I am pretty sure the attribute will be an easier sell for standards, just as it was for FB.

Reply:
Sounds good.